### PR TITLE
Update gns3-server and gns3-gui to 2.2.17

### PR DESCRIPTION
**Pull request by tsndqst**
Update gns3-server and gns3-gui to 2.2.17

### CODE REVIEW

**Comment by ahesford** [comment]
@the-maldridge: Do you approve of these maintainer changes?

**Comment by ericonr** [comment]
@ahesford The maintainer changes were discussed in another PR. #26971

File: srcpkgs/gns3-gui/template [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 short_desc="Graphical Network Simulator 3 - GUI"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
+maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gns3.com"
 distfiles="https://github.com/GNS3/${pkgname}/archive/v${version}.tar.gz"
-checksum=0f04fe1b3b328bab5ee80755ea90e18e0d25828fbf4a644f3fbec29189f19bc8
+checksum=@2200216046969000e6f076707d2ea3938a0777dc82e3d940cc41dc26c7fd46b5
 
 pre_build() {
```
Comment (ahesford) [inline srcpkgs/gns3-gui/template:18]: Please change `pre_build` to `post_patch`. `pre_build` only runs during the build stage, but sometimes it is nice to verify that the patching had the intended effect by running `xbps-src patch` without triggering the build stage.

File: srcpkgs/gns3-server/template [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
 short_desc="Graphical Network Simulator 3 - Server"
-maintainer="Michael Aldridge <maldridge@voidlinux.org>"
+maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gns3.com"
 distfiles="https://github.com/GNS3/gns3-server/archive/v${version}.tar.gz"
-checksum=4388974043680e877533564cb5a2efe2ed215a59a13abcabb3ac29d9b92f86e0
+checksum=@476c23fc41c0f92c816557f56e8a8184009b4cf0477c4d94156d9ff06db9256b
 
 pre_build() {
```
Comment (ahesford) [inline srcpkgs/gns3-server/template:18]: As above, please change `pre_build` to `post_patch`.

**Review by ahesford — CHANGES_REQUESTED** [review]
@ericonr: Thanks for pointing out the maintainer discussion.

**Pull request merged**